blip-tap.md

```diff
@@ -939,7 +940,10 @@ where:
 * `rfq_id` is a randomly generate 32-byte value to uniquely identify this RFQ
   request
 * `asset_id` is the asset ID of the asset the receiver intends to receive
-* `asset_amt` is the amount of units of said asset
+* `swap_type` is the type of swap the message sender is requesting. When the
+  `swap_type` is `1`, the sender is requesting a swap of their Tap asset for
+  BTC. When the `swap_type` is `2`, the sender is requesting a swap of their
+  BTC for the subject Tap asset.
```

Comment:
To support a more asset-agnostic/extensible way of referring to this direction, how about we modify the field to be: `in_asset` / `out_asset`

Reply:
@dstadulis But one of the assets is always forced by the blockchain LN runs on, right? Why specify that?

Reply:
@dstadulis 

> modify the field to be: `in_asset` / `out_asset`

I'm not sure I understand what you mean. Do you mean modify the `swap_type` field name? Or the `asset_id` field name? Or the way `swap_type` is documented?

When generalizing, we need to make sure we have all the necessary fields for it to make sense. Right now, we're only specifying one asset with `asset_id` and don't have any other field to mention another asset.

Reply:
PR which takes the in/out asset approach: https://github.com/Roasbeef/blips/pull/3

Reply:
This concept continues implementation here https://github.com/Roasbeef/blips/pull/3